src/renderers/webgl/WebGLTextures.js

```diff
@@ -1593,7 +1593,7 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 					const glFormat = utils.convert( texture.format, texture.encoding );
 					const glType = utils.convert( texture.type );
-					const glInternalFormat = getInternalFormat( texture.internalFormat, glFormat, glType, texture.encoding );
+					const glInternalFormat = getInternalFormat( texture.internalFormat, glFormat, glType, texture.encoding, renderTarget.isXRRenderTarget );
```

Comment:
Wait, I guess you mean `renderTarget.isXRRenderTarget === true`?
Otherwise you're passing `undefined` as a parameter that it's supposed to be a boolean.

Reply:
Maybe we can add `isXRRenderTarget` as a property in `WebGLRenderTarget`'s ctor with `false` as the default value?

Reply:
https://github.com/mrdoob/three.js/commit/bcdc5c5432dd338ab2b3fe9951d614711ba4f449

Reply:
> Maybe we can add `isXRRenderTarget` as a property in `WebGLRenderTarget`'s ctor with `false` as the default value?

Hmm, maybe not yet. The Quest Browser code seems to be in flux still...

Reply:
> [bcdc5c5](https://github.com/mrdoob/three.js/commit/bcdc5c5432dd338ab2b3fe9951d614711ba4f449)

@mrdoob thanks for fixing that up!